Cargo.toml

```diff
@@ -33,12 +33,12 @@ default = ["mimalloc"]
 [dependencies]
 tokio = { version = "1.24", features = ["macros", "rt", "rt-multi-thread", "sync"] }
 rand = "0.8"
-pyo3 = { version = "~0.17.3", features = ["extension-module", "abi3", "abi3-py37"] }
-datafusion = { version = "17.0.0", features = ["pyarrow", "avro"] }
-datafusion-expr = "17.0.0"
-datafusion-optimizer = "17.0.0"
-datafusion-common = { version = "17.0.0", features = ["pyarrow"] }
-datafusion-substrait = "17.0.0"
+pyo3 = { version = "0.18.0", features = ["extension-module", "abi3", "abi3-py37"] }
+datafusion = { git="https://github.com/apache/arrow-datafusion.git", branch="master", features = ["pyarrow", "avro"] }
```

Comment:
Could we pin to a specific rev here?

Reply:
Yes, I had it that way from the start and changed by to `branch=master` for some reason I no longer remember?? Anyway, just pinned it to a release.